composer.json

```diff
@@ -20,7 +20,7 @@
         "sebastian/exporter": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.0|^5.0"
+        "phpunit/phpunit": "^5.4"
```

Comment:
sebastianbergmann/phpunit@e9977fb0e8b290e1798184ebcec67daa54b2457e was first included in 5.4.3.